app/controllers/user/jobs_controller.rb

```diff
@@ -0,0 +1,30 @@
+class User::JobsController < UserController
+
+  def index
+    @jobs = current_user.jobs
```

Comment:
can you make a Professional model and use `current_professional` instead? We have two types of users, this will become confusing very fast